commons/src/test/resources/kapua-environment-setting.properties

```diff
@@ -66,6 +66,7 @@ commons.locale.zoneId=UTC
 #Locale ID (Default en_US). See {@link DayOfWeek} for more information about allowed settings for this parameter
 commons.locale.id=en_US
 #Local minimal days in first week (Deafault 7). See {@link WeekFields} for more information about allowed settings for this parameter
```

Comment:
We can set this parameter to 4 for now but we should discuss in one of the next scrum about which should be the more appropriate value since, setting to 7 days, all the weeks in the year should have the same length so, ideally, an equally distributed amount of messages in each data store index. 
Can you update also the comment (Default 7 -> Default 4)